website/project/__init__.py

```diff
@@ -92,6 +92,31 @@ def new_bookmark_collection(user):
     return collection
 
 
+def get_quickfiles_project_title(user):
```

Comment:
This file is an artifact of v1. To keep the code more cohesive, it might make sense to put these helpers in `osf.models.QuickFiles`.

Also, instead of `new_quickfiles_project(user)`, it would be more intuitive to implement a manager method, e.g. `QuickFiles.objects.create_for_user(user)`. It should error if the user already has quickfiles node.